### PR TITLE
Remove Prev/Next Buttons On Mobile

### LINKED ISSUE #322: Minor Update: Make Prev/Next Buttons Compatible with mobile
Get rid of the Previous/Next text for smaller/mobile screens. Just get rid of the text, not the buttons themselves, or the arrow icons.
image
You can look at things on mobile screens through the instructions in this ticket https://github.com/uclaacm/parcel-pointers/issues/300.

### PR DESCRIPTION
**Pull request by jenniferylee**
## Summary

Closes #322

Added media query to remove the prev/next buttons when the screen size shrinks/narrows to mobile width. This prevents the buttons from interfering with the text of the lessons. 

## Screenshots
Screenshot 2024-02-12 at 7 40 47 PM

### CODE REVIEW

File: src/styles/NavButtons.scss [hunk 1]
```diff
@@ -27,6 +27,9 @@ $header-height: 8vh;
     position: relative;
     top: 2px;
   }
+  @media screen and (max-width: 768px) {
```
Comment (Dankco) [inline src/styles/NavButtons.scss]: Let's just move these into the above &-text block css. We only want the text to disappear, not the entire button.

File: src/styles/NavButtons.scss [hunk 2]
```diff
@@ -52,6 +55,10 @@ $header-height: 8vh;
     right: 10px;
     top: 2px;
   }
+  // Hide buttons on mobile screens
+  @media screen and (max-width: 768px) {
```
Comment (Dankco) [inline src/styles/NavButtons.scss]: Let's just move these into the above &-text block css. We only want the text to disappear, not the entire button.

**Review by jenniferylee — COMMENTED** [review]
Moved to the &-text block css

**Review by Dankco — APPROVED** [review]
Thanks, looks good!

**Pull request merged**